PSOperations.xcodeproj/project.pbxproj

```diff
@@ -220,14 +244,8 @@
 				1D9B43391B6A934A008F7F18 /* NoCancelledDependencies.swift */,
 				1D9B433A1B6A934A008F7F18 /* MutuallyExclusive.swift */,
 				1D9B433B1B6A934A008F7F18 /* ReachabilityCondition.swift */,
-				1D9B433C1B6A934A008F7F18 /* CloudCondition.swift */,
-				1D9B433D1B6A934A008F7F18 /* PassbookCondition.swift */,
-				1D9B433E1B6A934A008F7F18 /* LocationCondition.swift */,
-				1D9B433F1B6A934A008F7F18 /* CalendarCondition.swift */,
-				1D9B43401B6A934A008F7F18 /* PhotosCondition.swift */,
-				1D9B43411B6A934A008F7F18 /* HealthCondition.swift */,
-				1D9B43421B6A934A008F7F18 /* RemoteNotificationCondition.swift */,
-				1D9B43431B6A934A008F7F18 /* UserNotificationCondition.swift */,
```

Comment:
Never mind. They're moved to a deprecated group and each class/struct was deprecated.